bluebrain/repo-bluebrain/packages/neurodamus/package.py

```diff
@@ -0,0 +1,431 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+import os
+import shutil
+from contextlib import contextmanager
+
+from llnl.util import tty
+
+from spack.package import *
+
+# Definitions
+_BUILD_NEURODAMUS_FNAME = "build_neurodamus.sh"
+WITH_CORENEURON = True
```

Comment:
I think you asked for it to not be a variant 😆 because we anyway always want to build with CoreNeuron. I kept a global because during tests I often want to start neuron only.

Reply:
OK, then this should be removed from the final version, then.